psydac/linalg/tests/test_kron_direct_solver.py

```diff
@@ -46,24 +45,22 @@ def kron_solve_seq_ref(Y, A, transposed):
     return X.reshape(Y.shape)
 # ...
 
-#===============================================================================
 # ... convert a 1D stencil matrix to band matrix
 def to_bnd(A):
 
-    dmat = dia_matrix(A.toarray())
+    dmat = dia_matrix(A.toarray(), dtype=A.dtype)
     la   = abs(dmat.offsets.min())
     ua   = dmat.offsets.max()
     cmat = dmat.tocsr()
 
-    A_bnd = np.zeros((1+ua+2*la, cmat.shape[1]))
+    A_bnd = np.zeros((1+ua+2*la, cmat.shape[1]),A.dtype)
```

Comment:
Please add an empty space after the comma.

Reply:
[86b06e9](https://github.com/pyccel/psydac/pull/273/commits/86b06e97fda7a028f2d6ec3f23b6c81d44eaf696)